rfc/drafts/mediachain-rfc-3.md

```diff
@@ -0,0 +1,449 @@
+
+# RFC 3: The V1 Indexer
+
+Status: DRAFT
+
+Author: [autoencoder](github.com/autoencoder)
+
+
+## Overview
+
+This document outlines a proposed `Indexer` system for Mediachain[1]. This system will perform media ingestion, media search and media deduplication. Specifically, this document focuses on the first version of the `Indexer` referred to here as `Indexer V1`.
```

Comment:
uhm, I don't think we should call it Indexer V1 -- just Indexer will do for the context.
Also perhaps it's better to call it first generation than first version?